exercises/practice/queen-attack/.approaches/introduction.md

```diff
@@ -0,0 +1,43 @@
+# Queen Attack Exercise
```

Comment:
Suggesting this change to be consistent with the headings used in other approaches.
```suggestion
# Introduction
```